components/badge/badge.css

```diff
@@ -0,0 +1,114 @@
+/* parent selector */
+
+.badge {
+  position: relative;
+}
+
+/* Icon badges */
+
+.icon {
```

Comment:
there could be any kinds of icons on a site, maybe icons.css a separate file?

Reply:
Have renamed classes to be specific to badges